firebase-vscode/common/messaging/protocol.ts

```diff
@@ -4,20 +4,16 @@
  */
 
 import { FirebaseConfig } from '../../../src/firebaseConfig';
-import { FirebaseRC } from "../firebaserc";
 import { User } from "../../../src/types/auth";
 import { ServiceAccountUser } from "../types";
+import { RCData } from '../rc';
 import { EmulatorUiSelections, RunningEmulatorInfo } from "./types";
 
 export interface WebviewToExtensionParamsMap {
   /**
-   * Ask extension for env variables
+   * Ask extension for initial data
    */
-  getEnv: {};
-  /**
-   * User management
-   */
-  getUsers: {};
+  getInitialData: {};
```

Comment:
This is bundling getEnv/getUsers/getChannels/getSelectedProject into one message since they're all requested by the webview upon init and not at any other time (they may be *pushed* by the extension at other times but not pulled by the webview)